vignettes/programming_strategy.Rmd

```diff
@@ -426,7 +426,7 @@ The documentation will be updated at:
 
 + the description level for a function, 
 
-    ```{r, eval=FALSE}
+    ```r
```

Comment:
Could you undo this change? It is intended that the R code is not executed. This allows developers to copy the code to the function they want to deprecate.